notebooks/IDC_segmentation_primer.ipynb

```diff
@@ -1,1927 +1 @@
-{
```

Comment:
<p>Perhaps it would be good to verify the downloads by displaying one of the images using itkWidgets.view()?   Would need to use ImageSeriesReader and GenerateSeriesFilenames...or update itk.imread to accept series_uid as an argument...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ImagingDataCommons/IDC-Tutorials/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
Should this wait until `imread` has been updated?

Reply:
Once `itk.imread` has been updated these notebooks can also be updated to use an example of this feature. This can be a future PR.